src/Mvc/test/WebSites/RazorBuildWebSite/RazorBuildWebSite.csproj

```diff
@@ -21,4 +20,6 @@
     <Reference Include="Microsoft.AspNetCore.Diagnostics" />
   </ItemGroup>
 
+  <Import Project="..\..\..\Mvc.Razor.RuntimeCompilation\src\targets\Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation.targets" />
```

Comment:
No `$(SolutionRoot)`?

Reply:
Ehh, unless something like helix requires it